tap_snowflake/__init__.py

```diff
@@ -101,17 +106,19 @@ def schema_for_column(c):
         result.type = ['null', 'string']
         result.format = 'time'
 
-    elif data_type in BINARY_TYPE:
-        result.type = ['null', 'string']
-        result.format = 'binary'
 
     elif data_type in SEMI_STRUCTURED_TYPES:
         result.type = ['null', 'string']
         result.format = 'semi_structured'
 
-    elif data_type in GEOGRAPHY_TYPE:
-        result.type = ['null', 'string']
-        result.format = 'geography'
+    # Symon: unsupported in target - mark as unsupported so that normal sync and s3 unload sync are consistent
+    # elif data_type in BINARY_TYPE:
+    #     result.type = ['null', 'string']
+    #     result.format = 'binary'
+
+    # elif data_type in GEOGRAPHY_TYPE:
+    #     result.type = ['null', 'string']
+    #     result.format = 'geography'
```

Comment:
this will make UI show unsupported type. prior to this change, our import will fail in target since we don't support binary, geography formats. with this change, we drop unsupported columns. 
<img width="1032" alt="image" src="https://github.com/symon-ai/pipelinewise-tap-snowflake/assets/86244437/3d368d9f-dcac-4c55-879c-1a6068f2ef45">

Reply:
before we comment out the unsupported types, were we able to import these columns?

Reply:
it fails in target-s3-parquet.